release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -65,6 +65,11 @@ This release adds improvements related to the following components and concepts.
 
 {op-system} now uses {op-system-base-full} 8.6 packages in {product-title} 4.11 and above. This enables you to have the latest fixes, features, and enhancements, as well as the latest hardware support and driver updates.
 
+[id="ocp-4-11-rhcos-redirector-url"]
+==== Updated {op-system} registry URL
+
+The redirector host name for downloading {op-system} boot images is now `rhcos.mirror.openshift.com`. You must configure your firewall to grant access to the boot images. See xref:../installing/install_config/configuring-firewall.html#configuring-firewall_configuring-firewall[Configuring your firewall for {product-title}] for more information.
```

Comment:
Recommend spelling out RHCOS in the first sentence
hostname one word per ISG
Consider putting "For more information," first, as that's preferred by the ISG

```suggestion
The redirector hostname for downloading {op-system-first} boot images is now `rhcos.mirror.openshift.com`. You must configure your firewall to grant access to the boot images. For more information, see xref:../installing/install_config/configuring-firewall.adoc#configuring-firewall_configuring-firewall[Configuring your firewall for {product-title}].
```